rotkehlchen/externalapis/manual_current_price.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants import ZERO
+from rotkehlchen.globaldb import GlobalDBHandler
+from rotkehlchen.interfaces import CurrentPriceOracleInterface
+from rotkehlchen.types import Price
+
+
+class ManualCurrentOracle(CurrentPriceOracleInterface):
+
+    def __init__(self) -> None:
+        super().__init__(oracle_name='manual current price oracle')
+
+    def rate_limited_in_last(self, seconds: Optional[int] = None) -> bool:
+        return False
+
+    def query_current_price(self, from_asset: Asset, to_asset: Asset) -> Price:
+        """Searches for a manually specified current price of the `from_asset`. If finds
+        the price, converts it to a price in `to_asset` and returns."""
+        from rotkehlchen.inquirer import Inquirer  # pylint: disable=import-outside-toplevel
+        manual_current_result = GlobalDBHandler().get_manual_current_price(
+            asset=from_asset,
+        )
+        if manual_current_result is None:
+            return Price(ZERO)
+        current_to_asset, current_price = manual_current_result
+        current_to_asset_price = Inquirer().find_price(
+            from_asset=current_to_asset,
+            to_asset=to_asset,
+        )
+        return Price(current_price * current_to_asset_price)
```

Comment:
why is this needed? To return the price in USD? Shouldn't you check that if to_asset == current to_asset skip this call?

Reply:
I don't think that this check is needed here since it happens in `Inquirer().find_price()`